src/GitVersionTfsTask/task.json

```diff
@@ -42,13 +51,22 @@
       "required": false,
       "helpMarkDown": "Additional arguments to send to GitVersion.exe",
       "groupName": "additional"
+    },
+    {
+      "name": "gitVersionPath",
```

Comment:
What's the use case for this? Since we deliver GitVersion as part of the TFS Task, I don't see the reason to use any other version in a different path.